### PR TITLE
core: frontend: App: improve messages

### PR DESCRIPTION
**Pull request by joaoantoniocardoso**

### CODE REVIEW

**Review by patrickelectric — COMMENTED** [review]
Thanks for the patch @joaoantoniocardoso but we are already late to create the 1.0.1 release, I'm happy to merge this PR after the review, but since you are changing the text a lot, I would also recommend to increase the number on `run_tour_version` for the next release.
Remember that you can test your changes with `yarn install && yarn serve`

File: core/frontend/src/App.vue [hunk 1]
```diff
@@ -327,7 +327,8 @@ export default Vue.extend({
         },
         {
           target: '#tour-center-hook',
-          content: 'Hope you enjoy your experience with BlueOS! Happy exploration!',
+          content: 'That's it! Now we want you to enjoy your experience with BlueOS! Also, don't forget to get in touch if
+          you need anything else to improve your journey! Happy exploring!',
```
Comment (joaoantoniocardoso) [inline core/frontend/src/App.vue:331]:
```suggestion
          content: `That's it! Now we want you to enjoy your experience with BlueOS! Also, don't forget to get in touch if
          you need anything else to improve your journey! Happy exploring!`,
```

**Comment by patrickelectric** [comment]
@joaoantoniocardoso can you fix the PR ?

**Review by patrickelectric — CHANGES_REQUESTED** [review]
Can you squash your changes and rebase over master ?

**Comment by patrickelectric** [comment]
> Thanks for the patch @joaoantoniocardoso but we are already late to create the 1.0.1 release, I'm happy to merge this PR after the review, but since you are changing the text a lot, I would also recommend to increase the number on `run_tour_version` for the next release. Remember that you can test your changes with `yarn install && yarn serve`

As requested, can you increase the `run_tour_version` number ?

**Review by patrickelectric — APPROVED** [review]

**Pull request merged**